net/i2pd/Makefile

```diff
@@ -36,45 +36,38 @@ define Package/i2pd
 endef
 
 define Package/i2pd/description
-	I2P (Invisible Internet Protocol) is a universal anonymous network layer. All
-	communications over I2P are anonymous and end-to-end encrypted, participants
-	don't reveal their real IP addresses.
+  I2P (Invisible Internet Protocol) is a universal anonymous network layer. All
+  communications over I2P are anonymous and end-to-end encrypted, participants
+  don't reveal their real IP addresses.
 endef
 
+TARGET_LDFLAGS+=-latomic
+
 define Package/i2pd/conffiles
 /etc/config/i2pd
 /etc/i2pd/i2pd.conf
 /etc/i2pd/tunnels.conf
-/etc/i2pd/tunnels.d/
+/etc/i2pd/tunnels.d/*
 endef
 
-define Build/Prepare
-	$(Build/Prepare/Default)
-	# race condition
-	cd $(PKG_BUILD_DIR); $(MAKE) mk_obj_dir
-endef
-
-TARGET_LDFLAGS+=-latomic
-
 define Package/i2pd/install
 	$(INSTALL_DIR) $(1)/usr/sbin
 	$(INSTALL_BIN) $(PKG_BUILD_DIR)/i2pd $(1)/usr/sbin
+	$(INSTALL_DIR) $(1)/etc/init.d
+	$(INSTALL_BIN) ./files/i2pd.init $(1)/etc/init.d/i2pd
 	$(INSTALL_DIR) $(1)/usr/share/i2pd
 	$(CP) $(PKG_BUILD_DIR)/contrib/certificates  $(1)/usr/share/i2pd
 	$(INSTALL_DIR) $(1)/etc/config
-	$(INSTALL_DATA) ./files/i2pd.config $(1)/etc/config/i2pd
+	$(INSTALL_CONF) ./files/i2pd.config $(1)/etc/config/i2pd
 	$(INSTALL_DIR) $(1)/etc/i2pd
-	$(INSTALL_CONF) $(PKG_BUILD_DIR)/contrib/i2pd.conf $(1)/etc/i2pd
+	$(INSTALL_DATA) $(PKG_BUILD_DIR)/contrib/i2pd.conf $(1)/etc/i2pd/i2pd.conf.example
 	$(SED) ' \
 		s/127.0.0.1/192.168.1.1/g; \
```

Comment:
Do we need to have this sed? It should not be necessary to modify example config, right?

Reply:
Can be removed, but I think better to show user that address must be 192.168.1.1 neither 127.0.0.1. What about second expression, that can be dropped.